### DISCUSSION TITLE
v1.2.4

### DISCUSSION THREAD

**Discussion by zippity21** [zippity21/Stardew_Valley_Regression_Mod#49] [Announcements]
## What's Changed
* Dialogue Addition by @TikkuTokku in https://github.com/zippity21/Stardew_Valley_Regression_Mod/pull/45
* Zippity21/hud alignment by @zippity21 in https://github.com/zippity21/Stardew_Valley_Regression_Mod/pull/48
* Note that the character dialogue has undergone a fair bit of change, including tonally, where the villagers are more encouraging than causing embarrassment .

## New Contributors
* @TikkuTokku made their first contribution in https://github.com/zippity21/Stardew_Valley_Regression_Mod/pull/45

**Full Changelog**: https://github.com/zippity21/Stardew_Valley_Regression_Mod/compare/v1.2.2...v1.2.4

<hr /><em>This discussion was created from the release <a href='https://github.com/zippity21/Stardew_Valley_Regression_Mod/releases/tag/v1.2.4'>v1.2.4</a>.</em>

**Comment by TikkuTokku** [comment]
Oh no, you put this out literally minutes before I added my new pull request!

**Reply by zippity21** [reply]
I just slipped your changes in 🥇

**Reply by TikkuTokku** [reply]
Thank you very much! I think I've burnt out a little on writing characters. If you know anyone who's willing to do the writing for a character they like, I'm happy to plug it into the code and make it work.

**Reply by zippity21** [reply]
You've done some awesome work, and I really appreciate it.

I always appreciate the creativity that people are able to contribute, and I hope that more people will continue in this vain.

I wonder how much it'd cost to commission some more writing... 🤔

**Comment by TikkuTokku** [comment]
I just downloaded the new release! I don't know if it's supposed to, but there's a zip of very release in the regression folder. That includes this release, which also has a zip of every release again, and again, and on and on. I deleted them from my folder.

**Reply by zippity21** [reply]
Oh, darn. That's a mistake on my end. I'll fix it.

**Reply by TikkuTokku** [reply]
It also for some reason doesn't include Gus or Penny. Jas's file is still updated with her dialogue, and they still have reactions in the en document, but their files aren't in the NPCs folder.

**Reply by zippity21** [reply]
Maybe I shouldn't be pushing releases at this point in the night, haha.

I've re-re-released it to include the additional dialogue.

That is something I should have caught from the pull request.
Whenever you add a new file that you want included in deployment, you have to add it to Regression.csproj, like this:

```
  <ItemGroup>
    <None Update="Regression Dialogue\Dialogue\NPCs\Penny.json">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>
  </ItemGroup>
```

**Reply by TikkuTokku** [reply]
I will keep that in mind for the future

I also Re-downloaded the fixed release and it looks great!
